src/main/java/uk/gov/hmcts/reform/civil/service/FeesPaymentService.java

```diff
@@ -76,20 +76,23 @@ public CardPaymentStatusResponse getGovPaymentRequestStatus(
             .externalReference(cardPaymentDetails.getPaymentGroupReference())
             .paymentFor(feeType.name().toLowerCase())
             .paymentAmount(cardPaymentDetails.getAmount());
+        log.info("CardPaymentStatusResponse {}", response);
+        if (paymentStatus.toUpperCase().equals("FAILED")) {
```

Comment:
Stringutils.equalsignorecase is better to make sure it goes through null check as well